colossalai/tensor/d_tensor/layout_converter.py

```diff
@@ -473,7 +473,7 @@ def _group_alive_check(cached_comm_action_sequence):
                 for process_group in used_process_groups:
                     try:
                         dist.get_rank(process_group)
-                    except RuntimeError as e:
+                    except (ValueError, RuntimeError) as e:
```

Comment:
In which case will it raise a value error?

Reply:
@ver217 I also got the same error, and from here it raises a ValueError whem group is not registered:
https://github.com/pytorch/pytorch/blob/1026b0f687af46a986eb6a78095c81f71284266e/torch/distributed/distributed_c10d.py#L923

which previously returned RuntimeError, changed by [this commit](https://github.com/pytorch/pytorch/commit/704b0b3c678aeb9739b40f654272d062acff2518#diff-61109d1cb2a0bd13fc51d678a82666295289da1ec0a1a694e73d9e8c28f51bdc)